xwiki-platform-core/xwiki-platform-skin/xwiki-platform-skin-skinx/src/main/java/com/xpn/xwiki/plugin/skinx/LinkExtensionPlugin.java

```diff
@@ -69,7 +69,7 @@ public String getLink(String link, XWikiContext context)
             result.append(StringEscapeUtils.escapeXml(entry.getValue().toString()));
             result.append("'");
         }
-        result.append("/>");
+        result.append("/>\n");
```

Comment:
Could it be interesting to have a comment explaining why this line break is needed there?

Reply:
It is not needed, it is just nicer for the HTML output as there is nothing else adding line breaks between these elements. This is now present in all of these skinx-plugins while previously only some of them had it. I haven't seen any comment in any of these having the line break.

Reply:
ok, if it's nothing more than having a nicer output, it's fine to leave it like that.